CONTRIBUTING.md

```diff
@@ -51,6 +51,29 @@ not necessary. Use one of these parameters to speed things up:
 * `-DskipTests` Compiles everything, runs no tests.
 * `-DskipITs` Compiles everything, runs unit tests, but no integration tests.
 
+If you do not (regularly) update the `ui/` module, you may want to use the properties
+`-Dui.disable-terser=true` and `-Dui.disable-es-lint=true` to speed up the build of the ui module by
+about 20 seconds.
+
+Hint: you can define a default set of properties in your `~/.m2/settings.xml` like this:
+```xml
+<settings>
+  <profiles>
+    <profile>
+      <id>nessie-props</id>
+      <activation>
+        <activeByDefault>true</activeByDefault>
+      </activation>
+      <properties>
```

Comment:
should we have a `skipUi` profile that sets those? I know we have `quickly`, but something to skip the UI only seems to be also helpful

Reply:
That becomes super tricky w/ maven profiles :(

Reply:
skipping UI may invalidate the `quarkus:run` task... At least IIRC that happened to be after a `clean install -Dquicky`. I believe the quarkus run task complained about missing UI modules... but I did not retest recently ;)

Reply:
I suppose we could still add `ui.disable-terser` and `ui.disable-es-lint` into a profile.